### ISSUE TITLE
Marktplace: Update thank you page skeleton

### ISSUE THREAD

**Issue by WBerredo** [Automattic/wp-calypso#74292]
This task will focus on the main changes designed in the figma link 5ddl1WFTEnaIatL2WlZp4k-fi-2459%3A3382

**Comment by epeicher** [comment]
This linked task will focus on the main theme card
- https://github.com/Automattic/wp-calypso/issues/73995

**Comment by gcsecsey** [comment]
This issue is currently blocked by https://github.com/Automattic/wp-calypso/issues/74300

Thread p1678890006521139-slack-C02JPCHEKDY

**Comment by gcsecsey** [comment]
To avoid conflicts, we've discussed with @WBerredo that in this issue we'll only address the style changes based on the FIgma designs, and we'll create a separate issue for handling text changes.
